src/main/java/frc/robot/RobotContainer.java

```diff
@@ -303,7 +304,8 @@ private void testCommands() {
    * @return the command to run in autonomous
    */
   public Command getAutonomousCommand() {
-    return autoBuilder.createAutoCommand();
+    // Always stow the arm before any auto
+    return new MoveArm(m_arm, ArmPose.StowedPose).andThen(autoBuilder.createAutoCommand());
```

Comment:
This is the one thing I did change from before - this will ensure the arm is stored before Auto runs. (We will need to make sure the MoveArm command actually finishes properly, otherwise the rest of auto won't wun)